boards/arm/nucleo_g474re/arduino_r3_connector.dtsi

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2019 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	arduino_header: connector {
+		compatible = "arduino-header-r3";
+		#gpio-cells = <2>;
```

Comment:
Need to add `gpio-map-mask` and `gpio-map-pass-thru` properties.

Reply:
@erwango and/or @pabigot can help with how these should be set.

Reply:
```
            gpio-map-mask = <0xffffffff 0xffffffc0>;
            gpio-map-pass-thru = <0 0x3f>;
```
for all cases.  See https://docs.zephyrproject.org/latest/guides/dts/index.html#gpio-nexus-nodes

Reply:
adding gpio-map-mask and gpio-map-pass-thru properties to boards/arm/nucleo_g474re/arduino_r3_connector.dtsi